### PR TITLE
Fix #57 - Modify var and spelling in conf

### LINKED ISSUE #57: Resolve typos and spelling in conf
There are a few typos in the full.conf, most of which are only spelling related typos, however, there is a variable name typo in:
- `mr_latenct` should be `mr_latency`

### PR DESCRIPTION
**Pull request by mbeacom**

### CODE REVIEW

File: conf/full.conf [hunk 1]
```diff
@@ -101,7 +101,7 @@ stats {
     # default: 0
     network         0;
     # the device name to stat the disk iops.
-    # ignore the device of /proc/diskstats if not configed.
+    # ignore the device of /proc/diskstats if not configured.
```
Comment (winlinvip) [inline conf/full.conf:104]: 👍

File: conf/full.conf [hunk 2]
```diff
@@ -142,15 +142,15 @@ http_api {
         allow_update        off;
     }
 }
-# embeded http server in srs.
+# embedded http server in srs.
```
Comment (winlinvip) [inline conf/full.conf:145]: 👍

File: conf/full.conf [hunk 3]
```diff
@@ -264,20 +264,20 @@ vhost scope.vhost.srs.com {
     # if off, all request access denied.
     # default: on
     enabled         off;
-    
+
     # whether enable min delay mode for vhost.
-    # for min latence mode:
+    # for min latency mode:
```
Comment (winlinvip) [inline conf/full.conf:269]: 👍

File: conf/full.conf [hunk 4]
```diff
@@ -484,33 +484,33 @@ vhost atc.srs.com {
...
     play {
         mw_latency      350;
     }
-    
+
     # @see publish.srs.com
     publish {
         mr          on;
-        mr_latenct  350;
+        mr_latency  350;
```
Comment (winlinvip) [inline conf/full.conf:496]: 👍

File: conf/full.conf [hunk 5]
```diff
@@ -484,33 +484,33 @@ vhost atc.srs.com {
...
     }
 }
 
-# the vhost for min delay, donot cache any stream.
+# the vhost for min delay, do not cache any stream.
```
Comment (winlinvip) [inline conf/full.conf:500]: 👍

File: conf/full.conf [hunk 6]
```diff
@@ -561,12 +561,12 @@ vhost publish.srs.com {
         #       memory(buffer) = latency * kbps / 8
         # for example, latency=500ms, kbps=3000kbps, each publish connection will consume
         #       memory = 500 * 3000 / 8 = 187500B = 183KB
-        # when there are 2500 publisher, the total memory of SRS atleast:
+        # when there are 2500 publisher, the total memory of SRS at least:
         #       183KB * 2500 = 446MB
-        # the value recomment is [300, 2000]
+        # the recommended value is [300, 2000]
```
Comment (winlinvip) [inline conf/full.conf:566]: 👍

File: conf/full.conf [hunk 7]
```diff
@@ -594,12 +594,12 @@ vhost refer.anti_suck.com {
...
         all           github.com github.io;
         # refer for publish clients specified.
-        # the common refer is not overrided by this.
+        # the common refer is not overridden by this.
         # if not specified this field, allow all.
         # default: not specified.
         publish   github.com github.io;
         # refer for play clients specified.
-        # the common refer is not overrided by this.
+        # the common refer is not overridden by this.
```
Comment (winlinvip) [inline conf/full.conf:602]: 👍

File: conf/full.conf [hunk 8]
```diff
@@ -975,12 +975,12 @@ vhost with-hls.srs.com {
         hls_ts_file     [app]/[stream]-[seq].ts;
         # whether use floor for the hls_ts_file path generation.
         # if on, use floor(timestamp/hls_fragment) as the variable [timestamp],
-        #       and use enahanced algorithm to calc deviation for segment.
+        #       and use enhanced algorithm to calc deviation for segment.
```
Comment (winlinvip) [inline conf/full.conf:978]: 👍

File: conf/full.conf [hunk 9]
```diff
@@ -1131,7 +1131,7 @@ vhost dvr.srs.com {
         #       segment,session apply it.
         # default: ./objs/nginx/html/[app]/[stream].[timestamp].flv
         dvr_path        ./objs/nginx/html/[app]/[stream].[timestamp].flv;
-        # the duration for dvr file, reap if exeed, in seconds.
+        # the duration for dvr file, reap if exceed, in seconds.
```
Comment (winlinvip) [inline conf/full.conf:1134]: 👍

File: conf/full.conf [hunk 10]
```diff
@@ -1179,15 +1179,15 @@ vhost ingest.srs.com {
         input {
             # the type of input.
             # can be file/stream/device, that is,
-            #   file: ingest file specifies by url.
-            #   stream: ingest stream specifeis by url.
+            #   file: ingest file specified by url.
```
Comment (winlinvip) [inline conf/full.conf:1182]: 👍

**Comment by winlinvip** [comment]
Appreciate for your work, thanks and give lots of confidence to me. I hope to improve my English if possible~

**Comment by mbeacom** [comment]
@winlinvip I didn't mean for my PR to seem crude! I just noticed the random adjustments as I was looking through your code, while I try to add to the HLS functionality.  I'm very eager to see how this project evolves.

**Comment by winlinvip** [comment]
I means it's really good PR.

**Comment by winlinvip** [comment]
I have merge it to go-oryx and I will merge to SRS. Thanks~
😄

**Comment by winlinvip** [comment]
@mbeacom Do not support HLS in go-oryx, because I find it's possible to support HLS+, which remux the RTMP to HLS on edge(while HLS is remux RTMP on origin).

**Comment by winlinvip** [comment]
https://github.com/ossrs/srs/commit/892591f9a6200f7de5e0a67235cb774c18246f79
Thanks~ 😃

**Comment by mbeacom** [comment]
@winlinvip So, you are transmuxing the RTMP to HLS on the edge?  That's great.

**Comment by winlinvip** [comment]
Yep, we have done it in our private product, it's ok to transmux on the edge. And we have public the solution(without code), use 302 or variant HLS to redirect the original HLS request to a new url with id. For example, when user request http://server/app/stream.m3u8, redirect to http://server/app/stream.m3u8?id=xxx. And each ts in m3u8 will be suffixed with the query with the id. So we can create a logic stream for HLS, and we can serve it by the same http handle, which transmux the RTMP to HLS.

The HLS+ demo: https://notehub.org/s5qiw

**Comment by winlinvip** [comment]
For exmaple, for the first request of HLS:

```
winlin:srs-plus winlin$ curl http://ossrs.net:8083/show/livestream.m3u8
Redirect to http://ossrs.net:8083/show/livestream.m3u8?shp_uuid=eba98df08622f770e0dcfa9000afcb45&shp_ts=1461293515678&shp_cid=133048
&shp_pid=2465&shp_sip0=182.92.80.26&shp_sip1=182.92.80.26&domain=ossrs.net
```

We will 302 the request to a new url, then:

```
winlin:srs-plus winlin$ curl 'http://ossrs.net:8083/show/livestream.m3u8?shp_uuid=eba98df08622f770e0dcfa9000afcb45&shp_ts=1461293515678&shp_cid=133048
&shp_pid=2465&shp_sip0=182.92.80.26&shp_sip1=182.92.80.26&domain=ossrs.net'
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-ALLOW-CACHE:YES
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-TARGETDURATION:2
#EXT-X-DISCONTINUITY
#EXTINF:1.527, no desc
livestream-0.ts?shp_uuid=eba98df08622f770e0dcfa9000afcb45&shp_ts=1461293515678&shp_cid=133048
&shp_pid=2465&shp_sip0=182.92.80.26&shp_sip1=182.92.80.26&domain=ossrs.net
&shp_seqno=0
```

When request the m3u8 with id, all ts suffixed with id, and the ts is generated dynamically for `this logical connection` with small duration(1.5s).

Then the second piece:

```
winlin:srs-plus winlin$ curl 'http://ossrs.net:8083/show/livestream.m3u8?shp_uuid=eba98df08622f770e0dcfa9000afcb45&shp_ts=1461293515678&shp_cid=133048
&shp_pid=2465&shp_sip0=182.92.80.26&shp_sip1=182.92.80.26&domain=ossrs.net'
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-ALLOW-CACHE:YES
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-TARGETDURATION:2
#EXT-X-DISCONTINUITY
#EXTINF:1.527, no desc
livestream-0.ts?shp_uuid=eba98df08622f770e0dcfa9000afcb45&shp_ts=1461293515678&shp_cid=133048
&shp_pid=2465&shp_sip0=182.92.80.26&shp_sip1=182.92.80.26&domain=ossrs.net
&shp_seqno=0
#EXTINF:1.530, no desc
livestream-1.ts?shp_uuid=eba98df08622f770e0dcfa9000afcb45&shp_ts=1461293515678&shp_cid=133048
&shp_pid=2465&shp_sip0=182.92.80.26&shp_sip1=182.92.80.26&domain=ossrs.net
&shp_seqno=1
```

HLS+ provides low latency and the same domain for RTMP and HLS+, it's very useful.

**Comment by mbeacom** [comment]
@winlinvip That is FANTASTIC!  I just went through all of the documentation that you linked to and provided.  Google's translator has come a long way!

Is there anything I can do to help contribute to the project?  I don't want to start writing out something that doesn't align with your vision for go-oryx.

Thanks!

**Comment by winlinvip** [comment]
@mbeacom I think you can help to correct the English wiki of SRS, which writen by my broken English... You can contact with me by Skype winterserver, and you will know the direction of SRS/oryx. What's about this advise?

**Comment by mbeacom** [comment]
@winlinvip That sounds good!  Let me know when will work best for you and we can discuss it!  Are there any specific sections you'd like me to review in the meantime?

**Comment by winlinvip** [comment]
Let's talk on Skype

**Comment by winlinvip** [comment]
My Skype is winterserver

**Pull request merged**